handlers/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+# handler file for system restart
+- name: start scylla-server
```

Comment:
Can all the functionality be moved to just the "restart handle", can set state to enabled, ect.. in there?   Most opensource playbook I've seen have just restart handle, especially looking at long term infra where changes needs to really call a restart vs start.

Reply:
Done, using restart handler